scripts/prepare_sector_network.py

```diff
@@ -2058,8 +2056,9 @@ def add_agriculture(n, costs):
     co2 = (
         nodal_energy_totals.loc[nodes, "agriculture oil"]
         * 1e6
+        / 8760
         * costs.at["oil", "CO2 intensity"]
-    )
+    ).sum()

```

Comment:
Doesn't the /8760 and .sum() cancel each other out?

Is it for consistency or am I missing something here?

Reply:
It does not cancel out, because the /8760 transforms it to hourly values (like in `p_set` above) and the `.sum()` sums up the values over the different nodes.